Base/usr/share/man/man7/kernel_cmdline.md

```diff
@@ -0,0 +1,129 @@
+## Name
+
+Kernel cmdline - boot arguments to control system functionality
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
+functionality can be used to disable or enable features in the OS entirely, but
+can sometimes be used also to change a feature's behavior without disabling it
+entirely.
+
+Some of the boot parameters require a value after it. In that case, you will
+need to write them like this:
+
+```
+param1=value ...
+
```

Comment:
There's still an extra empty line here :(

Reply:
Oops, I'll fix it :)